cfpb/consumerfinance.gov
# MegaMenu: Add passive listener to touchstart event

See background in https://web.dev/uses-passive-event-listeners/?utm_source=lighthouse&utm_medium=cli and https://github.com/WICG/EventListenerOptions/blob/gh-pages/explainer.md

<img width="809" alt="Screen Shot 2021-12-14 at 3 11 04 PM" src="https://user-images.githubusercontent.com/704760/146073704-71abcb75-64ed-4845-9576-74b082ccc5ae.png">

We do have a preventDefault in the triggerClick event handler (click event) https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/js/modules/behavior/FlyoutMenu.js#L223, but not in the handleTouchStart event handler.

## Additions

- Add passive listener to `touchstart` event.

## How to test this PR

1. `gulp clean && gulp build` should pass. The menu should still work. I don't see a difference versus production, but this'll satisfy lighthouse.
